example/package.json

```diff
@@ -1,49 +1,24 @@
 {
-  "name": "react-bubble-ui-example",
-  "homepage": ".",
-  "version": "0.0.0",
-  "private": true,
+  "name": "example",
+  "version": "1.0.0",
+  "main": "index.js",
+  "license": "MIT",
   "scripts": {
-    "start": "node ../node_modules/react-scripts/bin/react-scripts.js start",
-    "build": "node ../node_modules/react-scripts/bin/react-scripts.js build",
-    "test": "node ../node_modules/react-scripts/bin/react-scripts.js test",
-    "eject": "node ../node_modules/react-scripts/bin/react-scripts.js eject",
-    "predeploy": "npm run build",
-    "deploy": "gh-pages -d build"
+    "start": "parcel index.html",
```

Comment:
When I checked out this PR, running 'npm run start' under the example directory gives this error in the browser. 
<img width="685" alt="Screen Shot 2022-06-18 at 1 57 35 PM" src="https://user-images.githubusercontent.com/38799390/174452744-26715351-3a0b-4794-b427-7b53c21ae534.png">

Reply:
> When I checked out this PR, running 'npm run start' under the example directory gives this error in the browser. <img alt="Screen Shot 2022-06-18 at 1 57 35 PM" width="685" src="https://user-images.githubusercontent.com/38799390/174452744-26715351-3a0b-4794-b427-7b53c21ae534.png">

I think the example just came with a template, It was not updated